test/unit_tests/apps/youth-employment-success/js/reducers/route-option-reducer-spec.js

```diff
@@ -1,4 +1,5 @@
-import createRoute from '../../../../../../cfgov/unprocessed/apps/youth-employment-success/js/models/route';
+import { createRoute
+} from '../../../../../../cfgov/unprocessed/apps/youth-employment-success/js/models/route';
```

Comment:
I'd probably make this:

```
import { 
  createRoute
} from '../../../../../../cfgov/unprocessed/apps/youth-employment-success/js/models/route';
```

So it follows how we generally format object literals.

Reply:
I made the change and put import of createRoute on a separate line